docs/writing-stories/introduction.md

```diff
@@ -83,7 +83,7 @@ Use the _named_ exports of a CSF file to define your component’s stories. We r
 <!-- prettier-ignore-end -->
 
 <div class="aside">
-💡 The recommendation mentioned above also applies to other frameworks not only React.
+💡 The recommendation mentioned above also applies to other frameworks other than React.
 </div>
```

Comment:
@Nightvision53 care to elaborate on this? Looking at this the sentence is actually almost correct, what can be added here to improve the docs is a comma after frameworks, turning this into:
"The recommendation mentioned above also applies to other frameworks, not only React.".

Reply:
Okay, that will work too.